c2corg_ui/externs/appx.js

```diff
@@ -84,3 +84,186 @@ appx.BootstrapSliderValues;
  * }}
  */
 appx.auth.RequestChangePassword;
+
+
+/**
+ * @typedef {{
+ *   routes : Array<Object>,
+ *   waypoints : Array<Object>,
+ *   images : Array<Object>,
+ *   users : Array<Object>,
+ *   recent_outings : Array<Object>,
+ * }}
+ */
+appx.DocumentAssociations;
```

Comment:
Nope.

For instance for a waypoint document, you have ``all_routes`` but not ``routes``. For an outing document you have ``routes`` but not ``all_routes``.
For a waypoint document you have ``waypoint_parents`` and ``waypoint_children`` but not ``waypoints``. The contrary for a route document.